docs/site/migration/tables/lb3-only-commands.html

```diff
@@ -0,0 +1,104 @@
+<head>
+  {% include analytics.html %}
+</head>
+
+<table width="750">
+  <thead>
+    <tr>
+      <th width="200">Commands</th>
+      <th>Description</th>
+      <th>Workaround in LB4</th>
+    </tr>
+  </thead>
+  <tbody>
+
+    <tr>
+      <td>
+        <code>lb bluemix</code>
+      </td>
+      <td> Create a new application destined in Bluemix
+      </td>
+      <td> We have several tutorials on how to deploy a LoopBack 4 application to the IBM Cloud: <a href="https://loopback.io/doc/en/lb4/Deploying-to-IBM-Cloud.html">Deploying to Cloud Foundry on IBM Cloud</a>, <a href="https://loopback.io/doc/en/lb4/deploying_to_ibm_cloud_kubernetes.html">Deploying to Kubernetes on IBM Cloud</a>, <a href="https://loopback.io/doc/en/lb4/Appsody-LoopBack.html">Deploying to Kubernetes on IBM Cloud using Appsody</a>
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb property</code>
+      </td>
+      <td>
+        Adds properties to an existing model
+      </td>
+      <td> Open the model file, and manually add a new property.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb acl</code>
+      </td>
+      <td>
+        Adds acl settings to models
+      </td>
+      <td> Use the <a href="https://loopback.io/doc/en/lb4/Loopback-component-authentication.html">AuthenticationComponent</a> and <a href="https://loopback.io/doc/en/lb4/Loopback-component-authorization.html">AuthorizationComponent</a> to set up authentication and authorization manually.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb remote-method</code>
+      </td>
+      <td>
+        Adds a new remote method to a model
+      </td>
+      <td> Open the model's controller file and add a new method manually. Depending on the purpose of this new controller method, you may also need to a new method to the model's repository.
```

Comment:
the remote method command actually generates the strong-remoting style method definitions that can be transferred to openapi spec and prints out the dummy function, then you need to manually copy the dummy function to a model's js file. It doesn't write a function directly into the js file.
Could you provide link here https://loopback.io/doc/en/lb4/Controllers.html#writing-controller-methods and briefly mention "decorate the controller function with REST decorators"?

Reply:
![image](https://user-images.githubusercontent.com/6864736/80012817-e3828300-849b-11ea-9a80-ec6e610164a6.png)

Ok, so it adds it to the application, and not the model.

Reply:
> Could you provide link here https://loopback.io/doc/en/lb4/Controllers.html#writing-controller-methods and briefly mention "decorate the controller function with REST decorators"? 

Good idea.